maptivist/src/main/webapp/script.js

```diff
@@ -171,7 +99,8 @@ function searchBox(){
 function loadMarkersByBoundary() {
   fetch("/marker").then(response => response.json()).then((markers) => {
     var markersToDisplay = markers.filter(inBoundary);
-    for (let marker in markersToDisplay) {
+    var marker;
+    for (marker of markersToDisplay) {
```

Comment:
Does it work if you do:

`for (const marker of markersToDisplay) {`

as a single line?